gtsfm/averaging/translation/translation_averaging_base.py

```diff
@@ -33,6 +34,8 @@ def run(
         num_images: int,
         i2Ui1_dict: Dict[Tuple[int, int], Optional[Unit3]],
         wRi_list: List[Optional[Rot3]],
+        i2Ti1_priors: Optional[Dict[Tuple[int, int], PosePrior]] = None,
+        wTi_initial: Optional[Dict[int, PosePrior]] = None,
```

Comment:
You would not get a dictionary here. Atleast not how its currently present in the multiview optimizer. Can we use `wTi_priors=List[Optional[PosePrior]]` instead?

Reply:
now `Optional[List[Optional[Pose3]]]`